imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -556,9 +556,9 @@ func Test_fdFdstatSetFlags(t *testing.T) {
 	// Let's remove O_APPEND.
 	requireErrnoResult(t, wasip1.ErrnoSuccess, mod, wasip1.FdFdstatSetFlagsName, uint64(fd), uint64(0))
 	require.Equal(t, `
-==> wasi_snapshot_preview1.fd_fdstat_set_flags(fd=4,flags=0)
+==> wasi_snapshot_preview1.fd_fdstat_set_flags(fd=4,flags=)
 <== errno=ESUCCESS
-`, "\n"+log.String())
+`, "\n"+log.String()) // FIXME? flags==0 prints 'flags='
```

Comment:
that's fine IMHO because there are no flags and the formatted flags are pipe delimited. that said it is easy to change to special case none differently. @achille-roussel @ncruces any preferences on this one?

Reply:
this default applies to every flag type we have in wasi, btw.